main/options.c

```diff
@@ -302,6 +312,11 @@ static optionDescription LongOptionDescription [] = {
  {1,"       Output list of language mappings."},
  {1,"  --map-<LANG>=[+]map"},
  {1,"       Alternative version of --langmap option."},
+#ifdef HAVE_ICONV
+ {1,"  --output-encoding=encoding"},
+ {1,"      Specify encoding to write tags file. Default value is UTF-8,"},
+ {1,"      --output-encoding is provided. If not, nothing happen."},
```

Comment:
I guess the  `--output-encoding` mention in the description shuold rather be `--input-encoding`?

I guess I'd write something like this:  "The encoding to write the tag file in.  Defaults to UTF-8 if `--input-encoding` is specified, otherwise no conversion is performed."
or something like it.